app/integration_action.go

```diff
@@ -220,6 +224,9 @@ func (a *App) DoActionRequest(rawURL string, body []byte) (*http.Response, *mode
 		inURL.Path = path.Join("/", siteURL.Path, rawURLPath)
 		rawURL = inURL.String()
 	}
+	if strings.Contains(rawURL, "plugins/") {
```

Comment:
I think this could be wrong for non-relative URLs that aren't hosted internally, e.g. if configured to point at `https://some-service.example.com/plugins/endpoint`.

Should we just be replacing the block above that meant to do this but still ended up routing through the networking stack?

Reply:
Great catch.  The change will look like (combined with the block above)
```go
	rawURLPath := path.Clean(rawURL)
	if strings.HasPrefix(rawURLPath, "/plugins/") || strings.HasPrefix(rawURLPath, "plugins/") {
		return a.DoLocalRequest(rawURL, body)
	}
```